apps/aehttpclient/lib/client.ex

```diff
@@ -28,6 +28,28 @@ defmodule Aehttpclient.Client do
     end
   end
 
+  @spec get_raw_blocks({term(), binary(), binary()}) :: {:ok, term()} | {:error, binary()}
+  def get_raw_blocks({uri, from_block_hash, to_block_hash}) do
+    from_block_hash = Base.encode16(from_block_hash)
+    to_block_hash = Base.encode16(to_block_hash)
+    uri = uri <> "/raw_blocks?" <>
+            "from_block=" <> from_block_hash <>
+            "&to_block=" <> to_block_hash
+    case(get(uri, :raw_blocks)) do
```

Comment:
We can move the deserialization to the get function and call it here without making a case for what it returns, it does the error handling anyway.